draft-ietf-quic-tls.md

```diff
@@ -778,6 +778,21 @@ the connection that is hosted on stream 1.  This sequence number is not visible
 to QUIC.
 
 
+## Receiving Protected Packets
+
+Once an endpoint successfully receives a packet with a given packet number, it
+MUST discard all packets with higher packet numbers if they cannot be
+successfully unprotected with either the same key, or - if there is a key update
+- the next packet protection key (see {{key-update}}).  Similarly, a packet that
+appears to trigger a key update, but cannot be unprotected successfully MUST be
+discarded.
+
+Failure to unprotect a packet does not necessarily indicate the existence of a
+protocol error in a peer or an attack.  The truncated packet number encoding
```

Comment:
I thought that too, but a few factors lead me to think that this is still worth saying.  Where the error comes from doesn't change the outcome.  And severe reordering does happen on occasional packets.  Also, your current scheme for picking the truncation length fails if there is a sudden large increase in send rate; packets before the jump that are delayed until after the jump in rate might have shorter packet numbers and thus could be decoded incorrectly.